CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+* Added `conn.Ping(ctx)` method for check availability of `conn.Conn`
+* Refactored `cluster.Cluster.Get(ctx)` for return only available connection (instead return any connection from balancer)
```

Comment:
Refactored `cluster.Cluster.Get(ctx)` **to** return only available connection (instead of return**ing** any connection from balancer)